### PR TITLE
Monica adding file about San Francisco

### LINKED ISSUE #8: Monicanpm favorite places to eat
Let's add a document using: 
- [x] Create a branch
- [x] Add a new file 
- [x] Commit my file 
- [x] Open Pull Request 
- [ ] Discuss 
- [ ] Merge

### PR DESCRIPTION
**Pull request by monicanpm**
Resolves #8 by adding a file places to eat in San Francisco

### CODE REVIEW

File: eats/san francisco-ca.md--- [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### San Francisco, CA Great Places to Eat 
+
+* Pabu
+* Underground Ramen
```
Comment (rod11) [inline eats/san francisco-ca.md---]: Ramen!!! :ramen:

File: eats/san francisco-ca.md--- [hunk 2]
```diff
@@ -0,0 +1,4 @@
+### San Francisco, CA Great Places to Eat 
+
```
Comment (monicanpm) [inline eats/san francisco-ca.md---]: is this spelled correctly @monicanpm

**Comment by EmilyUna65** [comment]
yummy

**Comment by AMZod** [comment]
Sounds good! :+1:

**Comment by rod11** [comment]
:sheep:

**Pull request merged**